### PR TITLE
more added stuff

### PR DESCRIPTION
**Pull request by annlinnmar**
Added more stuff

### CODE REVIEW

File: src/newFile.txt [hunk 1]
```diff
@@ -1 +1 @@
-new 
\ No newline at end of file
+More added stuff :)
```
Comment (hera1234567) [inline src/newFile.txt]: ?

File: src/newFile.txt [hunk 2]
```diff
@@ -1,2 +1,6 @@
+
+More added stuff :)
```
Comment (hera1234567) [inline src/newFile.txt:2]: ?

File: src/newFile.txt [hunk 3]
```diff
@@ -1,2 +1,6 @@
...
+
 new sentence added :)
 even more new things :)
+
```
Comment (sluggerbit) [inline src/newFile.txt:6]: spotless apply

**Review by hera1234567 — CHANGES_REQUESTED** [review]

**Review by sluggerbit — CHANGES_REQUESTED** [review]
remove whitespace

**Pull request merged**